### PR TITLE
docs: rework plugins table + add logo assets & roadmap link

### PR DESCRIPTION
**Pull request by Salakar**
## Description

Quick PR to add logo image assets with a readme update to include the logo and also a link to the roadmap issue. Additionally reworked the plugin table, to not be a table.

Updated readme can be previewed here: https://github.com/FirebaseExtended/flutterfire/tree/%40salakar/readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,119 +1,229 @@
...
+ - [Cloud Messaging (`firebase_messaging`)](#firebase_messaging)
+ - [Cloud Storage (`firebase_storage`)](#firebase_storage)
+
+ - [Core (`firebase_core`)](#firebase_core)
+
+---
+
+### `firebase_admob`
+
+> [![firebase_admob][admob_badge_pub][admob_pub]
```
Comment (creativecreatorormaybenot) [inline README.md:43]:
```suggestion
> [![firebase_admob][admob_badge_pub]][admob_pub]
```

Comment (Salakar) [reply README.md:43]: Aye not finished yet :) still need to do all the other plugins

Comment (creativecreatorormaybenot) [reply README.md:43]: Haha no problem - just a coincidence that I looked at it 😄

Comment (Salakar) [reply README.md]: I should have left it as a daft so my bad :D thanks for reviewing though, appreciated

**Pull request merged**